### PR TITLE
fibb&calc

### PR DESCRIPTION
**Pull request by Yoka40**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,27 @@
...
+  else if(command=="multiply")
+    {
+      ret = std::to_string(first*second);
+    }
+  else if(command=="divide")
+    {
+      if(second == 0)
+      { ret = "Division by 0"; }
+      else
+      { ret = std::to_string(first/second);}
```
Comment (DamiXioR) [inline module1/homework/calculate/calculate.hpp:25]: hi, think this else is unnecessary

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+  if(sequence==0)  return 0;
+
+  else if(sequence<3)  return 1;
+  
+  else
```
Comment (DamiXioR) [inline module1/homework/fibonacci/fibonacci.hpp:23]: only sequence => 1 should return sequence, for 2 it's redundant, also you can skip last else

File: module1/homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -2,6 +2,27 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+
+ std::string ret = "Invalid data";
+ //jak zrobić warunek na int ?
+  if(command=="add")
```
Comment (piezol) [inline module1/homework/calculate/calculate.hpp:8]: Formatting of curly braces does not adhere to any standard i know of (and is inconsistent even within file).

File: module1/homework/fibonacci/fibonacci.hpp [hunk 4]
```diff
@@ -1,11 +1,25 @@
 #pragma once
 
+
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    
+    int fibonacci[sequence];
```
Comment (piezol) [inline module1/homework/fibonacci/fibonacci.hpp:6]: If you use a c-style array with size that is unknown at the compile time, you HAVE to allocate memory dynamically. 

Also, for the iterative algorithm any kind of array is not needed - you only ever need the last two numbers, so at the best case you would 'only' use too much memory here.

File: module1/homework/calculate/calculate.hpp (hunk 1 above)
Comment (ziobron) [reply module1/homework/calculate/calculate.hpp:25]: It's needed because there is no return in if.

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429, 1119, 1430], "points": [2, 5, 2, 6] }

**Pull request closed without merging**